solr/solr-ref-guide/modules/query-guide/pages/dense-vector-search.adoc

```diff
@@ -264,45 +264,113 @@ The `DenseVectorField` to search in.
 +
 How many k-nearest results to return.
 
-Here's how to run a KNN search:
+`preFilter`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: Depends on usage, see below.
+|===
++
+Specifies an explicit list of Pre-Filter query strings to use.
 
-[source,text]
-&q={!knn f=vector topK=10}[1.0, 2.0, 3.0, 4.0]
+`includeTags`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: none
+|===
++
+Indicates that only `fq` filters with the specified `tag` should be considered for implicit Pre-Filtering.  May not be combind with `preFilter`.
```

Comment:
Please ignore this suggestion in case it's wrong, I'm not a native speaker:
May not be combined -> Must not be combined?
May suggests 'optional'
Must implies a 'mandatory' condition? (because here it's not optional but mandatory I think)